Server/Controllers/AnswersController.cs

```diff
@@ -7,17 +7,17 @@
 namespace Quanda.Server.Controllers
 {
     /// <summary>
-    /// Kontroler obsługujący końcówki związane z odpowiedziami.
+    ///     Kontroler obsługujący końcówki związane z odpowiedziami.
     /// </summary>
     [Route("api/answers")]
     [ApiController]
     public class AnswersController : ControllerBase
     {
         /// <summary>
-        /// Repozytorium wykonujące akcje na odpowiedziach.
+        ///     Repozytorium wykonujące akcje na odpowiedziach.
         /// </summary>
         private readonly IAnswerRepository _repository;
-        private readonly int requestIdUser = 25; //future => Request.GetUser(); które będzie pobierane przy konkretnych zapytaniach do serwera HtttpRequest.Context.GetUser
+        private readonly int requestIdUser = 25; //HttpContext.Request.GetUserId()
```

Comment:
Suggestion for discussion - in pushed code use final form such as HttpContext.Request.GetUserId() and then add comment - something like //mock data: 25. When you use local version of code you can uncomment it and use it.